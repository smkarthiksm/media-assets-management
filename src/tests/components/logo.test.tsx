import '@testing-library/jest-dom';
import { renderWithProviders } from '../test-utility';
import LogoComponent from '../../components/logo/logo';

describe('LogoComponent', () => {
  test('should render', () => {
    const { container } = renderWithProviders(<LogoComponent />);
    expect(container).toMatchSnapshot();
  });
});
