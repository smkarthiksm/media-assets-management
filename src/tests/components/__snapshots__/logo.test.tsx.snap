// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoComponent should render 1`] = `
<div>
  <span
    class="logo"
  />
</div>
`;
